lib/src/widgets/dashboard_screen/summary.dart

```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
+                            ),
+                            Padding(
+                              padding: EdgeInsets.only(right: 38),
+                            ),
+                          ],
+                        ),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: <Widget>[
+                            Padding(
+                              padding: EdgeInsets.only(right: 24),
+                            ),
+                            Icon(
+                              Icons.fiber_manual_record,
+                              size: 12,
+                              color: SarakaColors.darkGray.withOpacity(0.2),
+                            ),
+                            Padding(
+                              padding: EdgeInsets.only(right: 3),
+                            ),
+                            Text(
+                              "Immature",
+                              overflow: TextOverflow.ellipsis,
+                              style: SarakaTextStyles.body,
+                              textAlign: TextAlign.center,
+                            ),
+                            Padding(
+                              padding: EdgeInsets.only(right: 16),
+                            ),
                           ],
-                          labelAccessorFn: (MatureCount row, _) =>
-                              '${row.title}',
-                          outsideLabelStyleAccessorFn: (MatureCount row, _) =>
-                              charts.TextStyleSpec(
-                                  color: charts.MaterialPalette.black),
                         ),
                       ],
-                      animate: true,
                     ),
-                  ),
+                  ],
+                ),
+                Padding(
+                  padding: EdgeInsets.only(bottom: 16),
                 ),
-                Center(
-                    child: Column(
-                  mainAxisAlignment: MainAxisAlignment.center,
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.end,
                   children: <Widget>[
-                    Text(
-                      totalCardsNumber,
-                      overflow: TextOverflow.ellipsis,
-                      style: SarakaTextStyles.body.copyWith(fontSize: 40),
-                      textAlign: TextAlign.center,
-                    ),
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.center,
-                      children: <Widget>[
-                        Padding(
-                          padding: EdgeInsets.only(left: 60.0),
+                    FlatButton(
+                      child: Container(
+                        width: 200,
+                        decoration: BoxDecoration(
+                          borderRadius: BorderRadius.all(Radius.circular(20.0)),
```

Comment:
Use `ShapeBorder` and `SuperellipseShape(borderRadius: BorderRadius.circular(24))` instead

Reply:
done.